docs/design-proposals/arch/arktos_scale_out.md

```diff
@@ -122,6 +124,11 @@ A resource manager contains the following components:
 * Active-standby Controller manager process
 * Active-standby scheduler
 
+### <a id="daemonset"></a>DaemonSet in Scale-out Arktos
+In the scale-out architecture, DaemonSet is only allowed for system tenant. A DaemonSet object resides on a Tenant Partition, in the TP's system tenant (by the priciple of no single "system partition", there exist multiple TP that have system tenants); DaemonSet controller and scheduler deployed on the TP create daemon Pods and assign then with proper node resource, respectively. Kubelet looks up the specific TP for API resources (e.g. configmap, secret) needed by the initialization of  system-tenanted daemon Pod.
```

Comment:
Even if users deploy Arktos in the traditional single cluster, the DS should be still allowed for system tenant ( or admin ) of the cluster. Scale-out only makes it more complex to manage.

Reply:
yes DS in system tenant should be allowed and supported in the so-called scale-up Arktos.
Good point about that clarification; I'll record it in the doc.

Reply:
doc restructured as separate file which covers scale-up and scale-up both.